calendar_af/models/calendar_appointment_suggestion.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are code similarities with calendar appointment, maybe break out to some common function if possible?